packages/backend/src/database/migrations/20220727122405_drop_issuer_fk.ts

```diff
@@ -0,0 +1,35 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+    if (await knex.schema.hasTable('openid_identities')) {
+        await knex.schema.alterTable('openid_identities', (tableBuilder) => {
+            tableBuilder.dropForeign('issuer');
+        });
+    }
+    await knex.schema.dropTableIfExists('openid_issuers');
+}
+
+export async function down(knex: Knex): Promise<void> {
+    if (!(await knex.schema.hasTable('openid_issuers'))) {
+        await knex.schema.createTable('openid_issuers', (tableBuilder) => {
+            tableBuilder.text('issuer').primary();
+        });
+        await knex('openid_issuers').insert({
+            issuer: 'https://accounts.google.com',
+        });
+    }
+    if (await knex.schema.hasTable('openid_identities')) {
+        await knex.schema.alterTable('openid_identities', (tableBuilder) => {
+            tableBuilder.dropPrimary();
+        });
+        await knex.schema.alterTable('openid_identities', (tableBuilder) => {
+            tableBuilder
+                .text('issuer')
+                .notNullable()
+                .references('issuer')
+                .inTable('openid_issuers')
+                .alter();
+            tableBuilder.primary(['issuer', 'subject']);
+        });
+    }
```

Comment:
is there restrictions in dropping the primary in the same `alter table` we set he new primary ?

Should we remove all the rows where the issuer doesn't match `https://accounts.google.com` ?